app/controllers/api/api.js

```diff
@@ -18,4 +19,66 @@ function authenticatedUser(req, res, next) {
 
 router.get('/secret', authenticatedUser, function (req, res, next) {
   res.json({message: "secret"});
+});
+
+//INDEX
+router.get('/api/bottles', authenticatedUser,function(req, res){
+  Bottle.find({user_id: req.user.id}, function(err, bottles){
+    if (err){
+      res.send("Something Wrong happened" + err)
+    } else {
+      res.send(bottles);
+    }
+  });
+})
+
+//CREATE
+router.post('/api/bottles', authenticatedUser, function (req, res){
+  var bottleParams = req.body.bottle;
+  bottleParams.user_id = req.user.id;
+  Bottle.create(bottleParams, function (err, bottles){
```

Comment:
this is creating a bottle. so intead of using `(err, bottles)` it should be `(err, bottle)`